content/install/users/user_admins.md

```diff
@@ -20,3 +24,35 @@ services:
 ```
 
 Please note that the usernames are case-sensitive and must match the exact values as returned from your version control system (e.g. GitHub).
+
+# Kill Stuck Jobs
+
+Kill jobs that cannot be cancelled:
+
+```
+drone build kill <repo/name> <build>
```

Comment:
This is able to be performed by a RepoAdmin now per https://github.com/drone/drone/pull/2327

Reply:
It would be great if someone could document that command ;)